Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM registry.access.redhat.com/ubi8/python-38
+
+WORKDIR /archspec
+COPY . .
+
+USER root
+
+RUN dnf update -y && dnf install git python38 python3-pip -y
+RUN pip3 install poetry tox-gh-actions coverage
+RUN pip3 install archspec
```

Comment:
Maybe try running it too?

`archspec cpu`

Reply:
why? this is not a test container

Reply:
To prove it works. ;)

I guess the point of this container isn't entirely clear to me...

Reply:
my impression is that this is a base image for people to use if they want to build something that runs archspec.

Reply:
yup, people like me that don't run Python on a daily basis, I just want to get the best of Spack/ArchSpec without having to install pip on my machine